src/SourceBuild/content/binary-tooling.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+### Usage: $0
+###
+###   Runs the binary tooling as needed for source-build scenarios. The tooling is responsible for
+###
+### Options:
+###   --binaries-keep-file <FILE>     Path to the file containing the list of allowed binaries to keep. Default is
+###                                   src/installer/src/VirtualMonoRepo/allowed-binaries.txt
```

Comment:
With this work (meaning the larger epic), should the allowed-binaries file eventually be moved into the SB content?  The synchronization won't need this anymore therefore it should be regular SB content.

Reply:
I think so. I'm unsure of where this should go however.